src/vklayer.c

```diff
@@ -946,9 +957,13 @@ static void vk_shtex_destroy_frame_objects(struct vk_data *data,
         struct vk_frame_data *frame_data =
             &queue_data->frames[frame_idx];
         bool *cmd_buffer_busy = &frame_data->cmd_buffer_busy;
+
```

Comment:
whitespace

Reply:
Removed extra whitespace in latest commit  584b757